src/test/java/org/junitpioneer/testkit/PioneerTestKitTests.java

```diff
@@ -74,8 +74,9 @@ void executeTestMethodWithParameterTypesSingleParameterTypeIllegalArgumentExcept
 				});
 
 				String expectedMessage = "methodParameterTypes must not be null";
-				assertThat(thrown).isInstanceOf(IllegalArgumentException.class);
-				assertThat(thrown.getMessage()).isEqualTo(expectedMessage);
+
+				org.assertj.core.api.Assertions.assertThat(thrown).isInstanceOf(IllegalArgumentException.class);
+				org.assertj.core.api.Assertions.assertThat(thrown.getMessage()).isEqualTo(expectedMessage);
```

Comment:
This could just be `Assertions.assertThat` if you do `import org.assertj.core.api.Assertions`